packages/cmrjs/tests/cmr-utils/test-cmr-utils.js

```diff
@@ -74,7 +75,7 @@ test.before(async (t) => {
   t.context.launchpadStub = sinon.stub(launchpad, 'getLaunchpadToken')
     .callsFake((config) => Promise.resolve(`${config.passphrase}-${config.api}-${config.certificate}`));
 
-  const bucketsJson = await readJsonFixture('./tests/fixtures/buckets.json');
+  const bucketsJson = await readJsonFixture(path.join(__dirname, '../fixtures/buckets.json'));
```

Comment:
Only reason for this change is to allow me to debug these tests using the VScode debugger. Otherwise the debugger would fail trying to load these relative paths